index.js

```diff
@@ -339,6 +344,31 @@ function addTest(testTitle, testCaseId, buildUrl) {
   }
 }
 
+function getAttachment(attachmentFolder, testTitle) {
+  const attachmentsFolder = path.join(process.cwd(), `${attachmentFolder}`)
+
+  if (fs.existsSync(attachmentsFolder)) {
+    const attachmentPath = getFiles(attachmentsFolder).find(filename => filename.includes(testTitle))
+     if (attachmentPath) {
+      return fs.readFileSync(attachmentPath).toString('base64')
+     } else {
+      console.log("Attachments with qTestId in name not found")
+     }
+  } else {
+    console.log("Attachments path not found")
+  }
+  return undefined
+}
+
+function getFiles (folderPath) {
+  const dirents = fs.readdirSync(folderPath)
+  const files = dirents.map((dirent) => {
+    const res = path.resolve(folderPath, dirent)
+    return fs.existsSync(res) && fs.lstatSync(res).isDirectory() ? getFiles(res) : res
+  })
+  return Array.prototype.concat(...files)
```

Comment:
are you converting files to array here? May it be simplified to `return [...files]`?